Tests/test_file_eps.py

```diff
@@ -57,6 +57,10 @@ def test_invalid_file(self):
         self.assertRaises(SyntaxError,
                           lambda: EpsImagePlugin.EpsImageFile(invalid_file))
 
+    def test_cmyk(self):
+        cmyk_image = Image.open("Tests/images/pil_sample_cmyk.eps")
+        self.assertEqual(cmyk_image.mode, "CMYK")
+
```

Comment:
Could you check for image correctness here? Or at least approximate correctness?